receiver/hostmetricsreceiver/README.md

```diff
@@ -0,0 +1,22 @@
+# Host Metrics Receiver
+
+The Host Metrics receiver generates metrics about the host system. This is intended to be used when the collector is
+deployed as an agent.
+
+The categories of metrics scraped can be configured under the `scrapers` key, e.g. to scrape cpu, memory and disk metrics
+(note not all of these are implemented yet):
+
+```
+hostmetrics:
+  scrape_interval: 10s
+  scrapers:
+    cpu:
+      report_per_process: true
+    memory:
+    disk:
```

Comment:
Should we include  a config for the OS?

Reply:
I don't think there's any need to add anything OS specific at this point (we may find a need for it later in which case we can add it then).

The overall plan is to keep the outputted metrics as close as possible across OSs. Any implementation code that's specific to an OS will be behind a build flag, so will only be included in the binaries for that OS. If a scraper is not currently supported for a given OS, then the collector should fail on startup if you try to include that scraper in the config.